security/security-core/src/main/java/io/camunda/security/configuration/OidcAuthenticationConfiguration.java

```diff
@@ -21,6 +21,7 @@ public class OidcAuthenticationConfiguration {
   private String jwkSetUri;
   private String authorizationUri;
   private Set<String> audiences;
+  private Set<TokenClaim> expectedClaims;
```

Comment:
:question: How does that work with yaml/env configuration?

Reply:
Good question, so with env configuration you can use a structure like:

```
CAMUNDA_SECURITY_AUTHENTICATION_OIDC_EXPECTEDCLAIMS_0_CLAIM=<name of claim>
CAMUNDA_SECURITY_AUTHENTICATION_OIDC_EXPECTEDCLAIMS_0_VALUE=<required value>
```

Or in yaml this would translate to

```
camunda:
  security:
    authentication:
      oidc:
        expected-claims:
          - claim: <claim name>
             value: <expected value>
```

We use this pattern pretty extensively in the management Identity code to map values to objects